pytorch_lightning/trainer/supporters.py

```diff
@@ -76,3 +76,17 @@ def _agg_memory(self, how: str):
                 return getattr(self.memory, how)()
             else:
                 return getattr(self.memory[:self.current_idx], how)()
+
+
+class Accumulator(object):
+    def __init__(self):
+        self.num_values = 0
+        self.total = 0
+
+    def accumulate(self, x):
```

Comment:
I'd rename this to `add_val` since from `accumulate` I'd expect what is currenty done by `mean`